models/field_definition_blackbox_test.go

```diff
@@ -1,18 +1,23 @@
-package models
+package models_test
 
 import (
 	"encoding/json"
 	"fmt"
 	"reflect"
 	"testing"
+
+	. "github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
 )
 
 func TestListFieldDefMarshalling(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
```

Comment:
So the whole idea was to decide at runtime if a test shall be executed, remember? I know that is a bit strange that a Unit test is a resource. In fact if you look at the Require() implementation you can already see that UnitTest is treated in a special way. But it is not subject of this PR to discuss this implementation. Feel free to open another PR.